RCAIDE/Library/Methods/Thermal_Management/Heat_Exchangers/Cross_Flow_Heat_Exchanger/append_cross_flow_heat_exchanger_conditions.py

```diff
@@ -0,0 +1,54 @@
+# RCAIDE/Library/Methods/Thermal_Management/Heat_Exchangers/Cross_Flow_Heat_Exchanger/append_cross_flow_heat_exchanger_conditions.py
+# 
+# Created:  Aug 2024, S. Shekar
+
+# ----------------------------------------------------------------------------------------------------------------------
+#  IMPORT
+# ----------------------------------------------------------------------------------------------------------------------
+import RCAIDE
+from RCAIDE.Framework.Mission.Common     import   Conditions
+
+
+# ---------------------------------------------------------------------------------------------------------------------- 
+#  append_cross_flow_heat_exchanger_conditions
+# ----------------------------------------------------------------------------------------------------------------------    
+def append_cross_flow_heat_exchanger_conditions(cross_flow_hex,segment,coolant_line,add_additional_network_equation):
+    
```

Comment:
function header

Reply:
None of the append_initial_conditions have any function headers, so I haven't included them to keep it consistent.

Reply:
Done